src/improvements/tasks/MultisigTask.sol

```diff
@@ -656,15 +656,41 @@ abstract contract MultisigTask is Test, Script {
         }
     }
 
-    /// @notice print the tenderly simulation link with the state overrides
-    function printTenderlySimulationLink(Action[] memory actions) internal view {
-        Simulation.StateOverride[] memory overrides = new Simulation.StateOverride[](1);
-        overrides[0] =
+    /// @notice print the tenderly simulation payload with the state overrides
+    function printTenderlySimulationPayload(Action[] memory actions) internal view {
+        Simulation.StateOverride[] memory stateOverrides = new Simulation.StateOverride[](1);
+        stateOverrides[0] =
             Simulation.overrideSafeThresholdOwnerAndNonce(parentMultisig, msg.sender, _getNonce(parentMultisig));
         bytes memory txData = _execTransationCalldata(
             parentMultisig, getMulticall3Calldata(actions), Signatures.genPrevalidatedSignature(msg.sender)
         );
-        Simulation.logSimulationLink({_to: parentMultisig, _data: txData, _from: msg.sender, _overrides: overrides});
+        
+        // Log the Tenderly JSON payload
+        string memory payload = string.concat(
```

Comment:
> Also, I would like an exception on `forge fmt` to maintain the json building code lines in MultisigTask.sol sane.

You can use `// forgefmt: disable-start` and `// forgefmt: disable-end` as needed